bot/constants.py

```diff
@@ -38,6 +38,7 @@
 SECONDS_PER_DAY = 86400
 CTF_CHAL_CATEGORIES = ("re", "pwn", "web", "crypto", "forensics", "other")
 APPROVAL_VALUES = ("PENDING", "ACCEPTED", "REJECTED")
+BLOCKED_DOMAINS = ['discord', 'groupme']
```

Comment:
discord.gg / discord.com / groupme.com 

That way when someone first joins and says: "Hello, excited to hang in the discord!" they don't get banned.

Reply:
Since we're only checking the substrings against links and not the entire message content, it won't trigger a ban in that case, right?

Reply:
Ah yeah missed the url regex